docs/src/getting_started.md

```diff
@@ -1,5 +1,13 @@
 # Getting Started
 
+[ChainRulesCore.jl](https://github.com/JuliaDiff/ChainRulesCore.jl) is a light-weight dependency for defining sensitivities for functions in your packages, without you needing to depend on ChainRules itself.
+
+[ChainRules.jl](https://github.com/JuliaDiff/ChainRules.jl) provides the full functionality, including sensitivities for Base Julia and standard libraries.
```

Comment:
Maybe also point out here that we load other sensitivities for other non-stdlib packages e.g. SpecialFunctions and nanmath on demand.

Reply:
sure, will add